filament/backend/src/opengl/ShaderCompilerService.cpp

```diff
@@ -263,8 +263,7 @@ ShaderCompilerService::program_token_t ShaderCompilerService::createProgram(
                     // compile the shaders
                     std::array<GLuint, Program::SHADER_TYPE_COUNT> shaders{};
                     compileShaders(gl,
-                            std::move(program.getShadersSource()),
-                            program.getSpecializationConstants(),
+                            program,
```

Comment:
I would like to keep the old code -- I didn't want to pass `program` directly into `compileShaders` because it's unclear what gets `std::move`'d at the call site.

Reply:
I see. Let me change it back.